### ISSUE TITLE
Figure out what to do with apps READMEs

### ISSUE THREAD

**Issue by germsvel** [poanetwork/mana#294]
Each app in the umbrella project has its own README. That's probably due to several projects being combined into a single umbrella. We should probably do something with those READMEs: either update them or remove them.

**Comment by hayesgm** [comment]
I like having them there if there's extra documentation we want to add about an app that's for semi-external consumption. That said, they should not just say "TODO" and if that's all we can think to put in them, then they should be removed. For instance, the CLI app now has usage notes in its README.md. And, for instance, the merkle trie app could have information about our implementation there.

**Comment by germsvel** [comment]
I agree. I think if there's good info we can put on them, we should keep them around. This might be good for @andogro to take a look?
